src/main/java/com/palpal/dealightbe/domain/item/presentation/ItemController.java

```diff
@@ -49,6 +49,13 @@ public ResponseEntity<ItemsRes> findAllForStore(@RequestParam Long memberId, @Pa
 		return ResponseEntity.ok(itemsRes);
 	}
 
+	@GetMapping("/members")
+	public ResponseEntity<ItemsRes> findAllForMember(@RequestParam double xCoordinate, @RequestParam double yCoordinate, @RequestParam String sortBy, @PageableDefault(size = 5, page = 0) Pageable pageable) {
```

Comment:
Pageable객체로 받고 계셔서 sortBy 따로 안받아도 쿼리스트링으로 `sort={기준}` 이런식으로 받을 수 있어요! 그렇게 하면서 더불어 정렬 기준의 디폴트 값도 넣어주면 좋을 것 같아요

```suggestion
	public ResponseEntity<ItemsRes> findAllForMember(@RequestParam double xCoordinate, @RequestParam double yCoordinate, @PageableDefault(size = 5, page = 0, sort={default}) Pageable pageable) {
```